product_docs/docs/edbcloud/beta/using_cluster/connecting_your_cluster/02_connect_on_premises.mdx

```diff
@@ -0,0 +1,93 @@
+---
+title: Connecting from Networks Outside Azure
+---
+
+
+If your applications are deployed on your on-premises environments, or on other Cloud providers instead Azure, you can set up a secure tunnel between your on premises network and the Azure virtual network. 
+
+There are a few options you can use.
+
+## Connecting Using Azure VPN Gateway (Point-to-Site)
+
+Azure VPN gateway provides you an encrypted tunnel to communicate
```

Comment:
This is a weird paragraph - can we delete it? It sounds like an ad for VPN Gateway. If we want to keep it, let's rewrite it to be more direct about the EDB Cloud specific benefits. The next paragraph already does that, though...